packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -175,17 +184,46 @@
 	}
 
 	.block-editor-block-mover__drag-handle.has-icon {
-		padding-left: 6px !important;
-		padding-right: 6px !important;
-		border-right: 1px solid $gray-900;
+		padding-left: 12px !important;
+		padding-right: 12px !important;
+	}
+
+	.block-editor-block-mover__move-button-container {
+		padding-left: 6px;
```

Comment:
It looks like this can be simplified a fair bit, and reduced. Right now there's 6px padding left and right inside the button, and the overall container, providing 12px padding total. Since space is at a premium, this might benefit from a small refactor, so it can become just 6px left and right, like so:
<img width="584" alt="Screenshot 2022-10-10 at 12 26 43" src="https://user-images.githubusercontent.com/1204802/194846192-7834e260-0b88-4bea-9f71-0859560e443d.png">

Here's a quick inspecting session, a calc-based width needs updating for this to work:
![tweak](https://user-images.githubusercontent.com/1204802/194846276-0d8fe562-b0df-4d05-b890-d126e61116cb.gif)